bundles/binding/org.openhab.binding.expire/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,31 @@
+Manifest-Version: 1.0
+Private-Package: org.openhab.binding.expire.internal
+Ignore-Package: org.openhab.binding.expire.internal
+Bundle-License: http://www.eclipse.org/legal/epl-v10.html
+Bundle-Name: openHAB Expire Binding
+Bundle-SymbolicName: org.openhab.binding.expire
+Bundle-Vendor: openHAB.org
+Bundle-Version: 1.9.0.qualifier
+Bundle-ManifestVersion: 2
+Bundle-Description: This is the Expire binding of the open Home Aut
+ omation Bus (openHAB)
+Import-Package: org.apache.commons.lang,
+ org.openhab.core.binding,
+ org.openhab.core.events,
+ org.openhab.core.items,
+ org.openhab.core.library.items,
+ org.openhab.core.library.types,
+ org.openhab.core.transform,
+ org.openhab.core.types,
+ org.openhab.model.item.binding,
+ org.osgi.framework,
+ org.osgi.service.cm,
+ org.osgi.service.component,
+ org.osgi.service.event,
+ org.slf4j
+Export-Package: org.openhab.binding.expire,
+ org.openhab.binding.expire.internal;x-internal:=true
```

Comment:
I have no idea what to have in MANIFEST.MF. I just used that one that was created by the archetype. Feel free to change it to the project's requirements ;-)